src/behaviors/lateral_driver_models.jl

```diff
@@ -2,7 +2,9 @@ abstract type LateralDriverModel end
 get_name(::LateralDriverModel) = "???"
 reset_hidden_state!(model::LateralDriverModel) = model # do nothing by default
 observe!(model::LateralDriverModel, scene::Scene, roadway::Roadway, egoid::Int) = model  # do nothing by default
-Base.rand(model::LateralDriverModel) = error("rand not implemented for model $model")
+Base.rand(model::LateralDriverModel) = rand(Random.GLOBAL_RNG, model)
```

Comment:
Same as above - we should be able to just have `LateralDriverModel <: DriverModel` and drop implementations of `rand`